.vtools-ref

```diff
@@ -0,0 +1 @@
+dev
```

Comment:
is there an advantage here over using naming conventions?

Reply:
mainly for testing. For example, if a redpanda PR needs functionality that's not yet in vtools' `dev`, a temporary commit can modify the reference so that it points to the desired branch/commit/etc..

Reply:
thinking more about this, there's an alternative that doesn't require a `.vtools-sha` file. I think it'd be best to remove the file and handle everything on the vtools side. I'll use this same PR to remove it once the vtools side is ready to handle this.